tests/features/cookie-consent-kit.feature

```diff
@@ -2,11 +2,28 @@
 Feature: Cookie consent kit.
   In order to show usage of cookie consent kit on the website
   As a anonymous user
+  I need to be able to configure the settings
   I want to be able to see the usage of CCK for remote videos.
+  I want to be able to see that the Cookie consent banner Js is loaded.
+
+  @BackupCookieConsentConfigs
+  Scenario: Create Webtools Cookie Consent settings
```

Comment:
Let's improve this test by logging out and checking that the script is present in the page when enabled, and that it varies based on the setting.

Reply:
done.